docs/authentication/embedded-sdk-add-passkeys.mdx

```diff
@@ -48,10 +48,18 @@ If your application is using the [NextAuth](https://next-auth.js.org) provider (
 
 - [x] **Authenticator Config**: Embedded SDK <mark>If they are required to point the Invoke URL to their application, shouldn't we say that it's a prerequisite? As mentioned below, the Invoke URL field is required and they can't leave this blank when creating their app in the Admin Console.</mark>
 
+  <p>
+    <font color="green">WM: MARK</font>
+  </p>
+
 - [x] **Grant Type**: Authorization Code flow
 
 - [x] **Client Type**: Confidential <mark>Can they have a **Public** Client Type?</mark>
```

Comment:
We can. "Use case details" is unclear to me. Is the purpose to highlight one specific use cases, or should we remove this line here and include details for both down below?

@ismailmustafa, if I recall correctly, there is a restriction on web. Is that correct, or did it just add steps to the user flow?

Reply:
Both Public and Confidential are acceptable. Public will not have a client secret and needs to use PKCE. I think our mobile examples are using public, or did at one point.

Reply:
That's a fair point. I think we initially used Confidential for the SDK Example Apps, and then used Public, but we probably didn't go back and update the docs.